Specs/Overlay Support Spec.md

```diff
@@ -0,0 +1,106 @@
+# Feature Specification: Overlays Support
+
+## Objective
```

Comment:
Shouldn't the spec include customer application and target audience?

Reply:
I have revised the objective statement to try capture the target audience.